release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -467,7 +467,32 @@ In the table, features are marked with the following statuses:
 === Removed features
 
 [id="ocp-4-6-bug-fixes"]
-== Bug fixes
+== Bug fixes 
+
+*Web console (Developer perspective)*
+
+* Previously, when you tried to delete a Knative application through the *Topology* view, a false positive error about a non-existing *Knative route* was reported. This issue is now fixed and the error is no longer displayed. link:https://bugzilla.redhat.com/show_bug.cgi?id=1866214[BZ#1866214])
+
+* Previously, Developer Console did not allow images from insecure registries to be imported. This bug fix adds a checkbox that allows user to use the insecure registries in the *Deploy image* form (link:https://bugzilla.redhat.com/show_bug.cgi?id=1826740[BZ#1826740])
+
+* When a user selected the *From Catalog* option to to create an application, the *Developer Catalog* displayed a blank page instead of a list of templates to create an application. This was caused when the 1.18.0 Jaeger operator was installed. This issue has now been fixed and the templates are displayed as expected. link:https://bugzilla.redhat.com/show_bug.cgi?id=1845279[BZ#1845279])
```

Comment:
s/to to/to
s/operator/Operator
Missing opening parenthesis before the link.

Reply:
```suggestion
* When a user selected the *From Catalog* option to create an application, the *Developer Catalog* displayed a blank page instead of a list of templates to create an application. This was caused when the 1.18.0 Jaeger Operator was installed. This issue has now been fixed and the templates are displayed as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1845279[BZ#1845279])
```